src/App.tsx

```
@@ -791,100 +872,154 @@ const App: Component = () => {
+          {/* Settings panel: expands up or down depending on bar position vs half screen height */}
+          <div
+            class="absolute left-0 right-0 overflow-hidden transition-[max-height] duration-300 ease-out border border-[var(--color-earthy-sage)]/30 bg-[var(--color-earthy-bg)]/95 backdrop-blur-sm shadow-lg"
+            classList={{
+              'max-h-0': !showContextPanel(),
+              'max-h-[70vh]': showContextPanel(),
+              'bottom-full rounded-t-2xl border-b-0': settingsExpandUp(),
+              'top-full rounded-b-2xl border-t-0': !settingsExpandUp(),
+            }}
+            onMouseEnter={cancelPanelClose}
+            onMouseLeave={panelMouseLeave}
+          >
```

Biome (2.3.14)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)